### PR TITLE
add rbac support in helm chart

### PR DESCRIPTION
**Pull request by yagonobre**
This change is [Reviewable](https://reviewable.io/reviews/luizalabs/teresa/364)

### CODE REVIEW

File: helm/README.md [hunk 1]
```diff
@@ -90,6 +90,8 @@ Parameter | Description | Default
 `build.limits.cpu` | CPU limit used by build POD  | `500m`
 `build.limits.memory` | Memory limit used by build POD | `1024Mi`
 `debug` | If true, print the stack trace on every panic/recover. | `false`
+`rbac.enabled` | If true, this configure teresa deployment to use rbac, for now
+it will use the `cluster-admin` role | `false`
```
Comment (drgarcia1986) [inline helm/README.md]: Really need that line break?

Comment (yagonobre) [reply helm/README.md]: not, updated

**Comment by yagonobre** [comment]
@aguerra @drgarcia1986 rebased

**Comment by drgarcia1986** [comment]
@yagonobre could you fix changelog?

**Comment by yagonobre** [comment]
@drgarcia1986 done

**Pull request merged**